packaging/rpm/make-microshift-app-images-rpm.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# First arg: file path containing user images per line
+# Second arg: container storage dir path
+# Third arg:  RPMBUILD_DIR
+
+RPMBUILD_DIR=$3
+_img_dir_=$2
+
+declare -a ARRAY
+
+#link filedescriptor 10 with stdin (standard input)
+exec 10<&0
+
+#stdin replaced with a file supplied as a first argument
+exec < $1
+let count=0
+
+#read user images into ARRAY
+while read LINE; do
+   ARRAY[$count]=$LINE
+   ((count++))
+done
+
+#restore stdin from file descriptor 10 then close filedescriptor 10
+exec 0<&10 10<&-
+
+#Generate microshift-app-images.spec 
+touch ./microshift-app-images.spec
+cat >./microshift-app-images.spec <<EOF
```

Comment:
We should probably propagate the changes from the other spec here. 

But I propose we merge, and then we keep working on the userland side.

Reply:
@mangelajo good idea. Could you open an issue for that please so we don't forget?